config/initializers/pipelines.rb

```diff
@@ -16,12 +16,8 @@
      (Rails.env.test? && ActiveRecord::Base.connection.table_exists?('automated_workflow_executions'))
 
     AutomatedWorkflowExecution.find_each do |automated_workflow_execution|
-      pipeline = Irida::Pipelines.instance.find_pipeline_by(
-        automated_workflow_execution.metadata['pipeline_id'],
-        automated_workflow_execution.metadata['workflow_version'],
-        'automatable'
-      )
-      automated_workflow_execution.disabled = pipeline ? !pipeline.executable : true
+      pipeline = automated_workflow_execution.workflow
+      automated_workflow_execution.disabled = pipeline.unknown? || !pipeline.automatable? || !pipeline.executable?
```

Comment:
Please add some simple test cases to verify the truth table for this.

Reply:
refactored and tests added in a011265